### ISSUE TITLE
dtype error in SP aquisition card driver when no averaging

### ISSUE THREAD

**Issue by lcontami** [Ecpy/ecpy_hqc_legacy#30]
```python
        else:
            for c in chs:
                b = buffers[c]
                b.reshape((-1, samples_per_record))
                b *= 1.9/65535
```

b is a int16 and should be float64.

**Comment by MatthieuDartiailh** [comment]
float32 is the smallest type that we could use here. Given that mean that we must copy the array the whole for loop could be replaced by a list comprehension.

**Comment by lcontami** [comment]
Should be fixed by #31

**Comment by MatthieuDartiailh** [comment]
Closed by #31
